docs/01-app/04-api-reference/03-file-conventions/layout.mdx

```diff
@@ -112,14 +95,214 @@ export default function RootLayout({ children }) {
 }
 ```
 
-- The root layout **must** define `<html>` and `<body>` tags.
-  - You should **not** manually add `<head>` tags such as `<title>` and `<meta>` to root layouts. Instead, you should use the [Metadata API](/docs/app/building-your-application/optimizing/metadata) which automatically handles advanced requirements such as streaming and de-duplicating `<head>` elements.
-- You can use [route groups](/docs/app/building-your-application/routing/route-groups) to create multiple root layouts.
-  - Navigating **across multiple root layouts** will cause a **full page load** (as opposed to a client-side navigation). For example, navigating from `/cart` that uses `app/(shop)/layout.js` to `/blog` that uses `app/(marketing)/layout.js` will cause a full page load. This **only** applies to multiple root layouts.
+## Examples
+
+### Nesting Layouts
+
+By default, layouts in the folder hierarchy are **nested**, which means they wrap child layouts via their `children` prop. You can nest layouts by adding `layout.js` inside specific route segments (folders).
+
+For example, to create a layout for the `/dashboard` route, add a new `layout.js` file inside the `dashboard` folder:
+
+<Image
+  alt="Nested Layout"
+  srcLight="/docs/light/nested-layout.png"
+  srcDark="/docs/dark/nested-layout.png"
+  width="1600"
+  height="606"
+/>
+
+```tsx filename="app/dashboard/layout.tsx" switcher
+export default function DashboardLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return <section>{children}</section>
+}
+```
+
+```jsx filename="app/dashboard/layout.js" switcher
+export default function DashboardLayout({ children }) {
+  return <section>{children}</section>
+}
+```
+
+If you were to combine the two layouts above, the root layout (`app/layout.js`) would wrap the dashboard layout (`app/dashboard/layout.js`), which would wrap route segments inside `app/dashboard/*`.
+
+The two layouts would be nested as such:
+
+<Image
+  alt="Nested Layouts"
+  srcLight="/docs/light/nested-layouts-ui.png"
+  srcDark="/docs/dark/nested-layouts-ui.png"
+  width="1600"
+  height="1026"
+/>
+
+### Creating multiple root layouts
+
+You can use [route groups](/docs/app/building-your-application/routing/route-groups) to create multiple root layouts.
+
+Navigating across multiple root layouts will cause a **full page load** (as opposed to a client-side navigation). For example, navigating from `/cart` that uses `app/(shop)/layout.js` to `/blog` that uses `app/(marketing)/layout.js` will cause a full page load.
+
+{/* TODO: Add Diagram */}
+
+### Adding metadata
+
+You should **not** manually add `<head>` tags such as `<title>` and `<meta>` to layouts. Instead, you can use the [Metadata API](/docs/app/building-your-application/optimizing/metadata) which automatically handles advanced requirements such as streaming and de-duplicating `<head>` elements.
```

Comment:
We should consider being less restrictive here, e.g. if you make it a client component, you need to manually add head tags. Instead, let's say "while you can add head tags, the Metadata API gives you type-safety, editor autocomplete, and blah blah"

Reply:
Maybe also "We recommend using this API, but..."